src/options/pages/marketplace/ServicesBody.tsx

```diff
@@ -28,15 +28,33 @@ import { type ServiceAuthPair } from "@/core";
 import { useAuthOptions } from "@/hooks/auth";
 import { useGetServicesQuery } from "@/services/api";
 import { joinName } from "@/utils";
+import Alert from "@/components/Alert";
 
 interface OwnProps {
   blueprint: RecipeDefinition;
 }
 
+const ErrorComponent: React.FC<{
+  error: string | string[] | undefined;
+  fieldIndex: number;
+}> = ({ error, fieldIndex }) => {
+  if (!Array.isArray(error)) {
+    return <></>;
```

Comment:
This looks a little bit weird.

```suggestion
    return null;
```